packages/frontend/app/utils/identity-comparison.ts

```diff
@@ -0,0 +1,15 @@
+import Identity from 'emberclear/models/identity';
+import { equalsUint8Array } from './uint8array-equality';
+
+export function identityArrayContains(arr: Identity[], identity: Identity): boolean {
```

Comment:
idk if this in pedantic, but thoughts on the name `identitiesInclude` (to kind of similar to the intent behind this smaller implementation: 

```ts
// similar to 'string'.includes(...)
export function identityIncludes(identities: Identity[], maybeHas: Identity): boolean {
  return identities.some(identity => equalsUint8Array(identity.publicKey, maybeHas.publicKey));
}
```

Reply:
I like that. It is a lot cleaner. I'll implement that.

Reply:
Addressed in eeb6ff1